hash_table.py

```diff
@@ -0,0 +1,42 @@
+class Hash_Table(object):
+    '''Class contains methods implementing a hash table.'''
+    def __init__(self, size=10):
+        self.hash_table = [None] * size
+
+    def get(self, key):
+        '''
+        Search the buckets in the hash table for the key. Return the value if
+        found. Raise an error if not.
+        '''
+        try:
+            for val in self.hash_table[self.hash(key)]:
+                if val == key:
+                    return val
```

Comment:
Does this work for implementations where the value to be stored is **not** the same as the key?